### PR TITLE
Add skill book variants, 2/?

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Content "Some another skill books got variations"
#### Describe the solution
Driving, launchers, lockpicking and stabbing books in this time - trying to make smaller, easier to review prs

### CODE REVIEW

**Comment by Kamayana** [comment]
Out of curiosity, are you keeping track of the ISBNs of the real-life books you're adding? Some of the book json files include the ISBN as a comment for the real-life books, which is a nice touch and indicates if a book is real or not. It's completely unnecessary and not even visible in-game, of course, so it's fine to skip unless it's trivially easy to add.

**Comment by GuardianDll** [comment]
Yeah, that's pretty easy, i can do it, but it seems i don't understand how to find them - `isbn-international.org` can't find anything i add no matter what i print

File: data/json/items/book/driving.json [hunk 1]
```diff
@@ -25,6 +54,78 @@
...
+      {
+        "id": "mag_cars",
+        "name": { "str": "Top Gear", "str_pl": "issues of Top Gear" },
+        "description": "Lots of articles about cars and driving techniques.",
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_1",
+        "name": { "str": "Automobile", "str_pl": "issues of Automobile" },
+        "description": "Nice looking magazine about automobile industry, with shiny images and long articles.  Dated 2019 year.",
```
Comment (TheShadowFerret) [inline data/json/items/book/driving.json:67]:
```suggestion
        "description": "A nice-looking magazine about the automobile industry, with shiny images and long articles.  Dated 2019.",
```

File: data/json/items/book/driving.json [hunk 2]
```diff
@@ -25,6 +54,78 @@
...
+        "id": "mag_cars_9",
+        "name": { "str": "Motor Trend", "str_pl": "issues of Motor Trend" },
+        "description": "Boring automobile magazine with big shiny label.",
+        "fun": -1,
+        "weight": 1
+      },
+      {
+        "id": "mag_cars_10",
+        "name": { "str": "Quatro Rodas", "str_pl": "issues of Quatro Rodas" },
+        "description": "It seems to be a car magazine, but you can undertand only images, as the text is wrote on another language.",
```
Comment (TheShadowFerret) [inline data/json/items/book/driving.json:123]:
```suggestion
        "description": "It seems to be a car magazine, but you can only understand the images, as the text is written in another language.",
```

~~Take that bilinguals~~

File: data/json/items/book/driving.json [hunk 3]
```diff
@@ -45,6 +146,44 @@
...
+      {
+        "id": "manual_driving_1",
+        "name": { "str": "Guide to Passing the Driving Test", "str_pl": "copies of Guide to Passing the Driving Test" },
+        "description": "A thick, heavy book about every thing you may be asked in your driving exam.  Biggest part of book contain theory quests with detailed answers, and many how-to's about driving overall.",
+        "weight": 1
+      },
+      {
+        "id": "manual_driving_2",
+        "name": { "str": "The official DVSA guide to driving", "str_pl": "copies of The official DVSA guide to driving" },
+        "description": "The book, packed with comprehensive information about driving, how to do it properly, and take a joy.  You wonder why is it here, as DVSA is a British organisation, and book has some information about specific british driving rules.",
```
Comment (TheShadowFerret) [inline data/json/items/book/driving.json:165]:
```suggestion
        "description": "The book, packed with comprehensive information about driving, how to do it properly, and take a joy.  You wonder why is it here, as DVSA is a British organization, and book has some information about specific British driving rules.",
```

File: data/json/items/book/driving.json [hunk 4]
```diff
@@ -64,6 +203,44 @@
...
+      {
+        "id": "textbook_anarch",
+        "name": { "str": "AAA Guide", "str_pl": "copies of AAA Guide" },
+        "description": "A tourist-centric guide to points of interest throughout the country.  This particular copy is apparently the Anonymous Anarchist's Annual: the cover conceals a wealth of ways to help stick it to The Man, along with plenty of advice for avoiding police.",
+        "weight": 1
+      },
+      {
+        "id": "textbook_anarch_1",
+        "name": { "str": "The Anarchist Cookbook", "str_pl": "copies of The Anarchist Cookbook" },
+        "description": "Classic illegal manual about making weapon and explosives.  The first page contain quote of original author, that describe it as \"a youthful indiscretion or mistake that can haunt someone during their early years or even longer\".",
```
Comment (TheShadowFerret) [inline data/json/items/book/driving.json:216]:
```suggestion
        "description": "Classic illegal manual about making weapons and explosives.  The first page contains a quote from its author William Powell, that describes it as \"a youthful indiscretion or mistake that can haunt someone during their early years or even longer\".",
```

File: data/json/items/book/driving.json [hunk 5]
```diff
@@ -64,6 +203,44 @@
...
+        "skill": "chemistry",
+        "required_level": 2,
+        "max_level": 5,
+        "fun": 0,
+        "weight": 1
+      },
+      {
+        "id": "textbook_anarch_3",
+        "name": { "str": "The Poor Man's James Bond", "str_pl": "copies of The Poor Man's James Bond" },
+        "description": "Weirdly named book with weird label, it contain a lot of information about how to make dangerous weapon and explosives in pretty dry language.",
```
Comment (TheShadowFerret) [inline data/json/items/book/driving.json:236]:
```suggestion
        "description": "A collection of American paramilitary knowledge throughout history, containing a lot of information about how to make dangerous weapons and explosives in pretty dry language.",
```

File: data/json/items/book/stabbing.json [hunk 6]
```diff
@@ -25,6 +75,62 @@
     "category": "manuals",
     "name": { "str": "FM 23-25 Army bayonet manual" },
     "description": "A hefty military field manual about bayonet usage and combat theory.",
+    "variants": [
+      {
+        "id": "manual_stabbing",
+        "name": { "str": "FM 23-25 Army bayonet manual", "str_pl": "copies of FM 23-25 Army bayonet manual" },
+        "description": "A hefty military field manual about bayonet usage and combat theory.",
+        "//": "idididididididid",
```
Comment (GuardianDll) [inline data/json/items/book/stabbing.json:83]:
```suggestion
```

File: data/json/items/book/launcher.json [hunk 7]
```diff
@@ -25,6 +60,66 @@
...
+        "//": "978-1472819529",
+        "weight": 1
+      },
+      {
+        "id": "manual_launcher_2",
+        "name": {
+          "str": "Catalogue of Standard Ordnance Items: Volume 3 & 4",
+          "str_pl": "copies of Catalogue of Standard Ordnance Items: Volume 3 & 4"
+        },
+        "description": "Giant, comprehensive list of every U.S. Army's weapon, equipment and ammunition that was used during World War II, with small, but detailed information about each of them.  Volume 3 describes light weapons like grenade launchers, recoilless rifles,  man-portable anti-tank and air-defense missiles and mortars, and volume 4 describe different heavy ammunition from 20mm to 16 inch hand and rifle grenades.",
```
Comment (GuardianDll) [inline data/json/items/book/launcher.json:83]:
```suggestion
        "description": "Giant, comprehensive list of every U.S. Army's weapon, equipment and ammunition that was used during World War II, with small, but detailed information about each of them.  Volume 3 describes light weapons like grenade launchers, recoilless rifles, man-portable anti-tank and air-defense missiles and mortars, and volume 4 describe different heavy ammunition from 20mm to 16 inch hand and rifle grenades.",
```

**Pull request merged**